examples/swarmion-full-stack/frontend/shared/src/assets/icons/assetAsString.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '*.png' {
```

Comment:
Hmm, is this the place for this declaration?

Reply:
@fargito where would you rather put it ?